### ISSUE TITLE
[Enhancement] Color - Decoration colors on chart

### ISSUE THREAD

**Issue by henrikvoetmand** [kirbydesign/designsystem#3824]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Decoration colors on chart

## Describe the solution you'd like
The yellow decoration color is changed to orange. And notification colors are changed.

    --kirby-danger: var(--kirby-decoration-color-red-50);
    --kirby-danger-contrast: #1c1c1c;
    --kirby-danger-shade: var(--kirby-decoration-color-red-60);
    --kirby-danger-tint: var(--kirby-decoration-color-red-30);
    --kirby-danger-color-brightness: light;
    --kirby-warning: var(--kirby-decoration-color-orange-40);
    --kirby-warning-contrast: #1c1c1c;
    --kirby-warning-shade: var(--kirby-decoration-color-orange-60);
    --kirby-warning-tint: var(--kirby-decoration-color-orange-30);
    --kirby-warning-color-brightness: light;
    --kirby-success: var(--kirby-decoration-color-green-30);
    --kirby-success-contrast: #1c1c1c;
    --kirby-success-shade: var(--kirby-decoration-color-green-70);
    --kirby-success-tint: var(--kirby-decoration-color-green-30);
    --kirby-success-color-brightness: light;

    --kirby-decoration-color-blue-10: #abe3f2;
    --kirby-decoration-color-blue-20: #88d7ec;
    --kirby-decoration-color-blue-30: #66cbe7;
    --kirby-decoration-color-blue-40: #43c0e1;
    --kirby-decoration-color-blue-50: #21b4dc;
    --kirby-decoration-color-blue-60: #1c98ba;
    --kirby-decoration-color-blue-70: #177c98;
    --kirby-decoration-color-blue-80: #126076;
    --kirby-decoration-color-blue-90: #0d4454;
    --kirby-decoration-color-green-10: #b9efd2;
    --kirby-decoration-color-green-20: #9de8c0;
    --kirby-decoration-color-green-30: #80e1ae;
    --kirby-decoration-color-green-40: #64db9b;
    --kirby-decoration-color-green-50: #47d489;
    --kirby-decoration-color-green-60: #3cb374;
    --kirby-decoration-color-green-70: #349C65;
    --kirby-decoration-color-green-80: #267149;
    --kirby-decoration-color-green-90: #1b5134;
    --kirby-decoration-color-purple-10: #e4d4e7;
    --kirby-decoration-color-purple-20: #d9c3de;
    --kirby-decoration-color-purple-30: #ceb1d4;
    --kirby-decoration-color-purple-40: #c3a0cb;
    --kirby-decoration-color-purple-50: #b88ec1;
    --kirby-decoration-color-purple-60: #9b78a3;
    --kirby-decoration-color-purple-70: #7f6285;
    --kirby-decoration-color-purple-80: #624c67;
    --kirby-decoration-color-purple-90: #463649;
    --kirby-decoration-color-red-10: #f7bec0;
    --kirby-decoration-color-red-20: #f3a4a6;
    --kirby-decoration-color-red-30: #f0898c;
    --kirby-decoration-color-red-40: #ec6f72;
    --kirby-decoration-color-red-50: #e95458;
    --kirby-decoration-color-red-60: #c5474a;
    --kirby-decoration-color-red-70: #a13a3d;
    --kirby-decoration-color-red-80: #7d2d2f;
    --kirby-decoration-color-red-90: #592021;
    --kirby-decoration-color-orange-10: #FFEAD8;
    --kirby-decoration-color-orange-20: #FFD8B5;
    --kirby-decoration-color-orange-30: #FFBB81;
    --kirby-decoration-color-orange-40: #FFAA61;
    --kirby-decoration-color-orange-50: #FF9D48;
    --kirby-decoration-color-orange-60: #D9731B;
    --kirby-decoration-color-orange-70: #B56118;
    --kirby-decoration-color-orange-80: #8E511B;
    --kirby-decoration-color-orange-90: #5A310D;

## Are there any additional context?
https://www.figma.com/design/bTihgLkrfmI3AkiaeEuUCF/Accessibility-design-assessment?m=auto&node-id=3045-6164&t=6zWoVRQgzrKyA1OC-1

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by Fuzzy3** [comment]
Currently we still expose the yellow color from the code as well as showcase it from our colors section in the cookbook. Consider if we should remove or keep the yellow color.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
